core/tests/ts-integration/src/test-master.ts

```diff
@@ -64,7 +64,12 @@ export class TestMaster {
             this.l2Provider.pollingInterval = 5000;
         }
 
-        this.mainWallet = new zksync.Wallet(suiteWalletPK, this.l2Provider, this.l1Provider);
+        // TODO: suiteWalletPK should be the same as the mainWalletPK used in context-owner.ts, otherwise, account won't have funds to operate with.
```

Comment:
Care to explain bit more about this comment?
Do we have an issue for this?

Reply:
Upon further inspecting how the testing works, it turns out that the individual wallet used for each test wasn't getting native erc20 funds needed, this is now done inside of `distributeL1BaseToken()`.